acceptance_tests/features/environment.py

```diff
@@ -38,18 +38,34 @@ def execute_collection_exercises():
     collection_exercise_controller.execute_collection_exercise(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54',
                                                                period='201812')
     logger.info('Waiting for collection exercises execution process to finish...')
-    time.sleep(240)
-
-
-def enrolment_setup():
-    valid_enrolment_code = database_controller.select_iac()
-    respondent_info = party_controller.register_respondent(email_address=Config.RESPONDENT_USERNAME,
-                                                           first_name='first_name',
-                                                           last_name='last_name',
-                                                           password=Config.RESPONDENT_PASSWORD,
-                                                           phone_number='0987654321',
-                                                           enrolment_code=valid_enrolment_code)
-    respondent_uuid = respondent_info['id']
-    django_oauth_controller.verify_user(respondent_info['emailAddress'])
-    case_id = database_controller.enrol_party(respondent_uuid)
-    case_controller.post_case_event(case_id, respondent_uuid, "RESPONDENT_ENROLED", "Respondent enrolled")
+    poll_database_for_iac(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201801')
+    poll_database_for_iac(survey_id='cb8accda-6118-4d3b-85a3-149e28960c54', period='201812')
+
+
+def poll_database_for_iac(survey_id, period):
+    collection_exercise_id = collection_exercise_controller.get_collection_exercise(survey_id, period)['id']
+    while True:
```

Comment:
nice one 👍

Reply:
Maybe worth looking bailing out at some point

Reply:
The timeout built into flask is already very close to how long it actually takes 👀